doc/chips/11.rst

```diff
@@ -0,0 +1,183 @@
+Alternative Initializer Features
+================================
+
+Status:
+  Rejected
+
+Authors:
+  Kyle Brady, Lydia Duncan, Michael Ferguson, Mike Noakes, Vassily Litvinov (for Syntax 2);
+  Vassily Litvinov (for Child Fields and Initialization Completion)
+
+Abstract
+--------
+
+This document contains the features we discussed as part of the
+initializer story and chose not to incorporate.
+
```

Comment:
You should consider adding an outline, especially since the rejected ideas could be much more distinct from each other, so a reader may wish to jump to one in particular